### PR TITLE
Create mydoc.md

### PR DESCRIPTION
**Pull request by ThanaMeri**

### CODE REVIEW

File: docs/mydoc.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+موقع Crowdin عبارة عن *منصة تعريب* تقدم خدمة ترجمة المواقع إلى عدة لغات بشكل احترافي وجماعي دون ضياع الجهد أو تداخله عن طريق تقاسم المهام بين المترجمين.
+
+- بدايةً قُم بإنشاء حساب في المنصة عن طريق التسجيل من خلال البريد الالكتروني (أو عبر وسائل التواصل الاجتماعي).
+  
+![1](https://user-images.githubusercontent.com/31999574/163883660-c793832f-43c9-45e4-836d-2d53e90ff318.png)
+  
+  ---
+  
+
+- سيتمً إرسال رسالة تأكيد إلى بريدك الالكتروني لتفعيل حسابك على الموقع.
```
Comment (BatoolMM) [inline docs/mydoc.md]: alignment should be from right to left - can you fix it, if possible, please?

**Review by BatoolMM — CHANGES_REQUESTED** [review]
Very beautiful review but can you fix the alignment, please?

File: docs/mydoc.md (hunk 1 above)
Comment (ThanaMeri) [reply docs/mydoc.md]: whenever I preview the code the alignment is correct but once I save it it goes to the left again(lines starts with *) so I had to remove bullet list after I had trying to solve it without any success.

**Review by BatoolMM — APPROVED** [review]
It looks amazing, thank you @ThanaMeri

**Pull request merged**